packages/turbo-utils/src/index.ts

```diff
@@ -1,8 +1,3 @@
 // utils
-export { default as getScopeFromArgs } from "./getScopeFromArgs";
```

Comment:
if anyone can remind me why I moved these over here in the first place I would love to hear that. turbo-ignore is their only consumer, so there's no reason they need to be abstracted out.